README.md

```diff
@@ -96,27 +96,22 @@ Optionally, change the `fail-build` field to `false` to avoid failing the build
 
 ### Action Inputs
 
-The only required key is `image`; all the other keys are optional. These are all the available keys to configure this action, along with its defaults:
+The only required key is `image` or `path`; all the other keys are optional. These are all the available keys to configure this action, along with its defaults:
 
 | Input Name          | Description                                                                                                                                                                                                                                                                                                    | Default Value |
 | ------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
-| `image`             | The image to scan                                                                                                                                                                                                                                                                                              | N/A           |
+| `image`             | The image to scan, this is mutually exclusive to `path`                                                                                                                                                                                                                                                        | N/A           |
+| `path`              | The file path to scan, this is mutually exclusive to `image`                                                                                                                                                                                                                                                   | N/A           |
```

Comment:
Just curious: This is not a functional change, it just wasn't documented before?

Reply:
Correct, it was [mentioned later in the readme](https://github.com/anchore/scan-action#directory-scanning) just not documented as a parameter for some reason.

Reply:
Okay, good catch 😄